api/src/main/java/com/stormpath/sdk/account/Account.java

```
@@ -298,6 +298,47 @@
     GroupMembership addGroup(Group group);
 
     /**
+     * Assigns this account to the specified Group represented by its (case insensitive) {@code name} or {@code href}
+     *
+     * <p><b>Immediate Execution:</b> Unlike other Account methods, you do <em>not</em> need to call {@link #save()}
+     * afterwards. This method will persist the changes in the backend immediately.</p>
+     *
+     * @param hrefOrName the href or name of the group to add.
+     * @return the new GroupMembership resource created reflecting the account-to-group association. Returns null if not group matching {@code hrefOrName} could be found.
+     *
+     * @since 1.0.RC5
+     */
+    GroupMembership addGroup(String hrefOrName);
+
+    /**
+     * Removes this account from the specified Group.
+     *
+     * <p><b>Immediate Execution:</b> Unlike other Account methods, you do <em>not</em> need to call {@link #save()}
+     * afterwards. This method will persist the changes in the backend immediately.</p>
+     *
+     * @param group the {@code Group} object from where the account must be removed.
+     * @return the Account object for method chaining
+     * @throws IllegalStateException if the Account does not belong to the specified group.
+     *
+     * @since 1.0.RC5
+     */
+    Account removeGroup(Group group);
```

Should throw an exception if the group is not assigned to the Account